app/src/js/components/DeleteCollection/DeleteCollectionModal.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Modal from 'react-bootstrap/Modal';
+import Button from '../Button/Button';
+import {
+  collectionName as collectionLabelForId,
+  deleteText
+} from '../../utils/format';
+
+class DeleteCollectionModal extends React.Component {
+  constructor (props) {
+    super(props);
+
+    this.handleCancel = this.handleCancel.bind(this);
+    this.handleConfirm = this.handleConfirm.bind(this);
+  }
+
+  handleCancel () {
+    this.props.onCancel();
+  }
+
+  handleConfirm () {
+    this.props.onConfirm();
+  }
+
+  render () {
+    return (
+      <Modal
+        dialogClassName="delete-collection-modal"
+        show={this.props.show}
+        onHide={this.handleCancel}
+        centered
+        size="md"
+        aria-labelledby="modal__delete-collection-modal"
+      >
+        <Modal.Header className="delete-collection-modal__header" closeButton />
```

Comment:
add the onCancel action to this close button

Reply:
Currently I'm clicking the x and nothing happens

Reply:
This is what I did with the bulk granules modal 

<Modal.Header className="bulk_granules-modal__header" closeButton onClick={this.cancel}></Modal.Header>

Reply:
It's working fine on this modal. It's the CollectionDeletedErrorModal and BatchAsyncCommand modals that are experiencing this issue. This should be able to bind with the handleCancel or handleClose function (whichever one we use). With react-bootstrap we should just use the closeButton and it knows when you declare the function before the render for both the "x" and the close/cancel button in the modal footer. I'll look at this.